docker-compose.yaml

```diff
@@ -19,6 +19,7 @@ services:
       # kubelet (default: 10250)
       - ${U7S_PORT_KUBELET}:${U7S_PORT_KUBELET}
       # flannel (default: 8472)
+      # This is a vxlan default, so we don't change it
```

Comment:
Why can't change?

Reply:
It requires reconfiguring the system which is out of scope for my expertise and what I intended to do for this PR. E.g., https://blog.svedr.in/posts/configuring-a-vxlan-interface-on-a-linux-machine/

If you want I can expose the variable for the expert user to change, but I can’t set up a testing environment for this.

Reply:
https://github.com/flannel-io/flannel/blob/v0.26.1/Documentation/backends.md#vxlan

> * `Port` (number): UDP port to use for sending encapsulated packets. On Linux, defaults to kernel default, currently 8472, but on Windows, must be 4789.

Reply:
So you are saying if I expose it and just add the value in the flannel yaml that is applied here:

```yaml
  net-conf.json: |
    {
      "Network": "10.244.0.0/16",
      "EnableNFTables": false,
      "Backend": {
        "Type": "vxlan"
      }
    }
```
it will just work? It doesn't require some change to the host system unrelated to kubernetes?

Reply:
Maybe use helm?
- https://github.com/flannel-io/flannel/issues/1426#issuecomment-1460407817

Reply:
Or just plain yq and jq

Reply:
@AkihiroSuda the helm chart values.yaml does not expose what we need, so I clone the repository and customize it for here, and have at least a start:

https://github.com/rse-ops/usernetes/pull/5

I think the missing piece is the changes to (kernel modules or the host network?) e.g., commands here: https://github.com/nehalineogi/azure-cross-solution-network-architectures/blob/main/advanced-linux-networking/linux-vxlan.md or here: https://docs.kernel.org/networking/vxlan.html

Could you help me to understand the minimum that I need to make this change?

Reply:
I think I figured this out.